types/qrcode-svg/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for qrcode-svg 1.0
+// Project: https://github.com/papnkukn/qrcode-svg
+// Definitions by: Eric Ferreira <https://github.com/ericbf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class QRCode {
+	constructor(content: string);
+	constructor(options: QRCode.Options);
```

Comment:
Can be `QRCode.Options | string` here

Reply:
I know it can be, but should it be, really? They are different parameters and have different names, that's why I used the override notation.

Reply:
Each overload has its own documentation – something impossible to do if they are combined.